portal-web/docroot/html/common/themes/session_timeout.jspf

```diff
@@ -11,35 +11,24 @@ String rememberMe = CookiesManagerUtil.getCookieValue(CookiesConstants.NAME_REME
 
 <c:if test="<%= !themeDisplay.isSignedIn() || Validator.isNull(rememberMe) %>">
 	<aui:script sandbox="<%= true %>">
-		AUI().use(
-			'liferay-session',
-			function () {
-				Liferay.Session = new Liferay.SessionBase(
-					{
+		import('/o/frontend-js-utils-web/__liferay__/session.js').then(
```

Comment:
Do we really want to use dynamic imports for all this or should we use a regular import from the `<aui:script>` tag as we do in other places? :thinking: 

IMO if we can do it from the tag it's better, because that allows for easier maintenance and code searching...

Reply:
Yeah that was my plan, just putting it like this for now to test against CI